packages/cli/src/cli.ts

```diff
@@ -178,6 +178,11 @@ Finish by saving the following secrets to your GitHub repository:
 `;
   }
 
+  if (argv._.includes('prepare')) {
+    await prepareCodebases(true);
+    return $`echo 'prepare'`;
```

Comment:
```suggestion
    return prepareCodebases(false);
```

Tem nada de `echo`  😄
Esse argumento booleano aí é o `isDev`, ele faz preparar o codebase SSR pra dev server, acho que não precisa aqui (e faria o setup dos testes demorar mais)